feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/TimetableScreenViewModel.kt

```diff
@@ -44,6 +44,9 @@ class TimetableScreenViewModel @Inject constructor(
     private val timetableUiTypeStateFlow: MutableStateFlow<TimetableUiType> =
         MutableStateFlow(TimetableUiType.List)
 
+    private val onBookmarkClickStateFlow: MutableStateFlow<Boolean?> =
```

Comment:
How about naming this bookmarkAnimationStartStateFlow? As the guideline says, ViewModel should provide states.